.github/workflows/main.yml

```diff
@@ -104,6 +104,12 @@ jobs:
       run: |
         mkdir publish
         dotnet run --property WarningLevel=0 --project .Lib9c.Tools/Lib9c.Tools.csproj -- action list > publish/all_action_type_ids.txt
+        dotnet run --property WarningLevel=0 --project .Lib9c.Tools/Lib9c.Tools.csproj -- action list --obsolete-only --json-path=publish/obsolete_action_types.json
+    - name: Build Dashboard
+      run: |
+        cd Dashboard
+        yarn && yarn build --outDir ../publish
+        cd ..
```

Comment:
The Dashboard can be deployed independently from the Lib9c's release schedule. How about creating a new job to deploy the Dashboard app?